src/Microsoft.AspNetCore.Testing/xunit/FrameworkSkipConditionAttribute.cs

```diff
@@ -33,24 +33,23 @@ private static bool CanRunOnThisFramework(RuntimeFrameworks excludedFrameworks)
                 return true;
             }
 
+#if NET451
             if (excludedFrameworks.HasFlag(RuntimeFrameworks.Mono) &&
                 TestPlatformHelper.IsMono)
             {
                 return false;
             }
 
-            if (excludedFrameworks.HasFlag(RuntimeFrameworks.CLR) &&
-                PlatformServices.Default.Runtime.RuntimeType.Equals("CLR", StringComparison.OrdinalIgnoreCase))
+            if (excludedFrameworks.HasFlag(RuntimeFrameworks.CLR))
             {
                 return false;
             }
-
-            if (excludedFrameworks.HasFlag(RuntimeFrameworks.CoreCLR) &&
-                PlatformServices.Default.Runtime.RuntimeType.Equals("CoreCLR", StringComparison.OrdinalIgnoreCase))
+#else
```

Comment:
Yes, but they package and build differently than we do so their versions end up in the right place. We don't have that same guarantee. What we are doing here is similar, and it might work, but it's not guaranteed to work.